charts/assets/issuer.yaml

```diff
@@ -1,7 +1,8 @@
+---
 apiVersion: cert-manager.io/v1
 kind: ClusterIssuer
```

Comment:
Can we switch to `Issuer`?

Reply:
Sure. Does it matter which one is used, or are you just thinking more on least-privilege reasons?

Reply:
>  just thinking more on least-privilege reasons?

:)